src/taipy/templates/taipy-default-template/hooks/post_gen_project.py

```diff
@@ -10,8 +10,174 @@
 # specific language governing permissions and limitations under the License.
 
 import os
+import shutil
 
 import taipy
 
+
+def handle_services(use_rest, use_core):
+    if use_core in ["YES", "Y"] or use_rest in ["YES", "Y"]:
+        # Write "import taipy as tp" at the third line of the import.txt file
+        with open(os.path.join(os.getcwd(), "sections", "import.txt"), "r") as import_file:
+            import_lines = import_file.readlines()
+            import_lines[1] += "import taipy as tp\n"
+        with open(os.path.join(os.getcwd(), "sections", "import.txt"), "w") as import_file:
+            import_file.writelines(import_lines)
+
+    if use_rest in ["YES", "Y"]:
+        with open(os.path.join(os.getcwd(), "sections", "import.txt"), "a") as import_file:
+            import_file.write("from taipy import Rest\n")
+        with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a") as main_file:
+            main_file.write("    rest = Rest()\n")
+
+    if use_core in ["YES", "Y"]:
+        # Import and run Core service if Rest service is not run yet
+        if use_rest not in ["YES", "Y"]:
+            with open(os.path.join(os.getcwd(), "sections", "import.txt"), "a") as import_file:
+                import_file.write("from taipy import Core\n")
+            with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a") as main_file:
+                main_file.write("    core = Core()\n")
+
+        # Create and submit the placeholder scenario
+        with open(os.path.join(os.getcwd(), "sections", "import.txt"), "a") as import_file:
+            import_file.write("\nfrom configuration import scenario_config\n")
+        with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a") as main_file:
+            main_file.write("    scenario = tp.create_scenario(scenario_config)\n")
+            main_file.write("    scenario.submit()\n")
```

Comment:
I would not generate some code that creates a scenario at the application run. This is not the standard way of creating a scenario. As a user, I would first remove these lines to implement my own method to create scenarios (either a GUI control or eventually check if a scenario exists before creating a new one...).

We can use a placeholder comment eventually
```
    # ##################################################################################################################\n
    # PLACEHOLDER: Initialize your data application here                                                               #\n
    #                                                                                                                  #\n
    # Example:                                                                                                         #\n
    # from configuration import scenario_config                                                                        #\n
    # if len(tp.get_scenarios()) == 0:                                                                                 #\n
    #     tp.create_scenario(scenario_config, name="Default Scenario")                                                 #\n
    # Comment, remove or replace the previous lines with your own use case                                             #\n
    # ##################################################################################################################\n
```